### PR TITLE
Feat: add sortable icon & aria-label on DetailsList column

### LINKED ISSUE #21767: User can't tell which columns are sortable in DetailsList
### Environment Information

- **Package version(s)**: v7.x,8.x,9.x
- **Browser and OS versions**: all

### Describe the issue:

For DetailsList columns which have tri-state sort (ascending, descending, and unsorted), there is no indication in the column header by default that the column is sortable when in the unsorted state. Specifically, the code which is the culprit is in [DetailsColumn.base.tsx](https://github.com/microsoft/fluentui/blob/c4912723213bb1a15b4af2f6c7962bc32064bf4a/packages/react/src/components/DetailsList/DetailsColumn.base.tsx#L168) where the sorting icon is not rendered when isSorted is falsey. This violates the [first principle of accessibility](https://www.w3.org/WAI/WCAG21/Understanding/intro#understanding-the-four-principles-of-accessibility) because the user has no way of knowing which columns are sortable vs which are not.

### Please provide a reproduction of the issue in a codepen:

[https://codepen.io/DavidParks8/pen/JjOOdBz](https://codepen.io/DavidParks8/pen/JjOOdBz) 
Ask yourself when the page loads whether or not the "Date Modified" column is sortable.

#### Actual behavior:

Sortable columns end up looking the same as non-sortable columns when the sortable columns are in an "unsorted" state. There is no extra icon in the column to indicate sortability.

#### Expected behavior:

Sortable columns in an unsorted state should show the fluentui up down arrow icon 
image
 to indicate they are sortable. When the IColumn is sortable but unsorted, there is not enough metadata in the IColumn to know that it is sortable. Currently, there is only isSorted, and isSortedDescending. There needs to be another field or extra metadata that indicates if the column can be sorted, and if so, an up/down arrow icon should be shown in the column header.

### Documentation describing expected behavior

[The four principles of accessibility from WCAG](https://www.w3.org/WAI/WCAG21/Understanding/intro#understanding-the-four-principles-of-accessibility)

### PR DESCRIPTION
**Pull request by pdrutherford**
## Current Behavior

A column that could be sorted (e.g., has an onColumnClick function that sets the isSorted/isSortedDescending properties on the column) does not provide any indicator that it is sortable. 

## New Behavior

A column that could be sorted can (backwards compatible, so not set by default) be made to show a sort icon (up + down arrow) and have an aria-label indicating the column could be sorted.

## Related Issue(s)

Fixes #21767

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d1112b254407f0f0c27119326ec86d2112bc6d70'>d1112b254407f0f0c27119326ec86d2112bc6d70</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-examples/src/react/DetailsList/DetailsList.CustomColumns.Example.tsx [hunk 1]
```diff
@@ -78,6 +78,13 @@ export class DetailsListCustomColumnsExample extends React.Component<{}, IDetail
     thumbnailColumn.ariaLabel = 'Thumbnail';
     thumbnailColumn.onColumnClick = undefined;
 
+    //indicate that all columns except thumbnail column can be sorted
+    columns.forEach((_column: IColumn, index: number) => {
+      if (columns[index].name) {
+        columns[index].isSortable = true;
+      }
+    });
```
Comment (DavidParks8) [inline packages/react-examples/src/react/DetailsList/DetailsList.CustomColumns.Example.tsx:86]: the columns array does not need to be accessed by index.
```
columns.forEach((column: IColumn) => {
  if (column.name) {
    column.isSortable = true;
  }
});
```

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/BaseButton-mount_base.html">794</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/BaseButton-mount.html">780</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Breadcrumb-mount_base.html">2380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Breadcrumb-mount.html">2407</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Checkbox-mount_base.html">1266</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Checkbox-mount.html">1260</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/CheckboxBase-mount_base.html">1026</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/CheckboxBase-mount.html">1084</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ChoiceGroup-mount_base.html">4079</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ChoiceGroup-mount.html">4076</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ComboBox-mount_base.html">832</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ComboBox-mount.html">837</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/CommandBar-mount_base.html">9151</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/CommandBar-mount.html">8991</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ContextualMenu-mount_base.html">10210</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ContextualMenu-mount.html">10139</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DefaultButton-mount_base.html">983</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DefaultButton-mount.html">1015</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DetailsRow-mount_base.html">3288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DetailsRow-mount.html">3205</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DetailsRowFast-mount_base.html">3147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DetailsRowFast-mount.html">3194</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DetailsRowNoStyles-mount_base.html">3118</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DetailsRowNoStyles-mount.html">3152</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Dialog-mount_base.html">1928</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Dialog-mount.html">1962</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DocumentCardTitle-mount_base.html">141</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/DocumentCardTitle-mount.html">149</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Dropdown-mount_base.html">2692</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Dropdown-mount.html">2858</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/FocusTrapZone-mount_base.html">1565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/FocusTrapZone-mount.html">1547</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/FocusZone-mount_base.html">1531</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/FocusZone-mount.html">1556</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/IconButton-mount_base.html">1524</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/IconButton-mount.html">1459</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Label-mount_base.html">298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Label-mount.html">304</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Layer-mount_base.html">2377</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Layer-mount.html">2526</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Link-mount_base.html">392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Link-mount.html">406</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/MenuButton-mount_base.html">1259</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/MenuButton-mount.html">1279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/MessageBar-mount_base.html">1836</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/MessageBar-mount.html">1704</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Nav-mount_base.html">2855</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Nav-mount.html">2854</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/OverflowSet-mount_base.html">925</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/OverflowSet-mount.html">967</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Panel-mount_base.html">1901</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Panel-mount.html">1891</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Persona-mount_base.html">877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Persona-mount.html">837</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Pivot-mount_base.html">1173</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Pivot-mount.html">1238</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/PrimaryButton-mount_base.html">1114</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/PrimaryButton-mount.html">1102</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Rating-mount_base.html">6546</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Rating-mount.html">6629</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/SearchBox-mount_base.html">1066</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/SearchBox-mount.html">1015</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Shimmer-mount_base.html">2029</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Shimmer-mount.html">2135</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Slider-mount_base.html">1531</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Slider-mount.html">1670</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/SpinButton-mount_base.html">4353</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/SpinButton-mount.html">4455</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Spinner-mount_base.html">368</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Spinner-mount.html">365</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/SplitButton-mount_base.html">2745</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/SplitButton-mount.html">2709</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Stack-mount_base.html">440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Stack-mount.html">401</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/StackWithIntrinsicChildren-mount_base.html">1951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/StackWithIntrinsicChildren-mount.html">1913</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/StackWithTextChildren-mount_base.html">4538</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/StackWithTextChildren-mount.html">4480</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/SwatchColorPicker-mount_base.html">9884</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/SwatchColorPicker-mount.html">9895</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/TagPicker-mount_base.html">2318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/TagPicker-mount.html">2335</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/TeachingBubble-mount_base.html">84885</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/TeachingBubble-mount.html">85672</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Text-mount_base.html">365</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Text-mount.html">375</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/TextField-mount_base.html">1202</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/TextField-mount.html">1235</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ThemeProvider-mount_base.html">1035</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ThemeProvider-mount.html">1013</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ThemeProvider-virtual-rerender_base.html">556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ThemeProvider-virtual-rerender.html">562</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1633</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1595</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Toggle-mount_base.html">694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/Toggle-mount.html">697</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/buttonNative-mount_base.html">114</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22947/perf-test/buttonNative-mount.html">109</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/DetailsList/DetailsList.types.ts [hunk 2]
```diff
@@ -433,6 +433,9 @@ export interface IColumn {
   /** If true, allow the column to be collapsed when rendered in justified layout. */
   isCollapsible?: boolean;
 
+  /** If true, column header will render a icon indicating column is sortable while unsorted */
```
Comment (DavidParks8) [inline packages/react/src/components/DetailsList/DetailsList.types.ts:436]:
```suggestion
  /** If true, column header will render an icon indicating the column is sortable while unsorted. */
```

File: packages/react/src/components/DetailsList/DetailsList.types.ts [hunk 3]
```diff
@@ -433,6 +433,9 @@ export interface IColumn {
...
+  isSortable?: boolean;
```
Comment (DavidParks8) [inline packages/react/src/components/DetailsList/DetailsList.types.ts:437]: I'm wondering if the fluent owners will have an issue with the naming of this prop. Technically, a column can be sortable even when isSortable is falsey. They may want it renamed to something like "showSortIconWhenUnsorted". @ThomasMichon will likely have the final say.

**Review by DavidParks8 — CHANGES_REQUESTED** [review]

File: packages/react/src/components/DetailsList/DetailsList.types.ts [hunk 4]
```diff
@@ -513,6 +516,12 @@ export interface IColumn {
    */
   sortDescendingAriaLabel?: string;
 
+  /**
+   * Accessible label for indicating that the list could be sorted by this column but isn't currently.
+   * This will be read after the main column header label.
+   */
+  sortableAriaLabel?: string;
```
Comment (DavidParks8) [inline packages/react/src/components/DetailsList/DetailsList.types.ts:523]: I don't see this PR rendering this anywhere. Did I miss something?

Comment (pdrutherford) [reply packages/react/src/components/DetailsList/DetailsList.types.ts:523]: Correct, and good catch! I'd missed where the actual aria-label was being set.

File: packages/react/src/components/DetailsList/DetailsList.types.ts (hunk 3 above)
Comment (pdrutherford) [reply packages/react/src/components/DetailsList/DetailsList.types.ts]: Concur, and changed to the name you suggested. Open to reverting if the simpler name is preferred even with the lack of specifity

**Review by spmonahan — APPROVED** [review]

File: packages/react/src/components/DetailsList/DetailsList.types.ts [hunk 5]
```diff
@@ -433,6 +433,9 @@ export interface IColumn {
   /** If true, allow the column to be collapsed when rendered in justified layout. */
   isCollapsible?: boolean;
 
+  /** If true, column header will render an icon indicating column is sortable while unsorted */
+  showSortIconWhenUnsorted?: boolean;
```
Comment (spmonahan) [inline packages/react/src/components/DetailsList/DetailsList.types.ts:437]: This could be `showUnsortedIcon` if you wanted a more concise name.

**Pull request merged**